Sources/Apollo/ApolloStore.swift

```diff
@@ -258,3 +301,48 @@ public final class ApolloStore {
     }
   }
 }
+
+internal extension NormalizedCache {
```

Comment:
It's probably not worth the trouble, but I wonder if we could use something like [Node's `promisify`](https://nodejs.org/api/util.html#util_util_promisify_original) to avoid writing these wrapper methods manually.

Reply:
Since I'd eventually like to remove our promises system I'm not entirely convinced making them easier to create is worth it. 😆